views/index.html

```diff
@@ -2,13 +2,15 @@
 {% block body %}
     <script src="{{ personaHost }}/include.js"></script>
     {% include 'userbar.html' %}
-    <script src="/bower/webmaker-login-ux/dist/min/webmakerLogin.min.js"></script>
     {% include 'tutorialbar.html' %}
     {% include 'friendlycode.html' %}
     {% block ssooverride %}
       <script id="ssooverride" src="/scripts/ssooverride.js"
-              data-csrf="{{ csrf }}"
-              data-login-hostname="{{ LOGIN_URL }}"
+              data-login-authHostname="{{ OAUTH_AUTHORIZATION_URL }}"
```

Comment:
Wouldn't that mean templating the js file itself? At some point a huge refactor is coming, and a lot of this will change.